internal_ws/ykbh/src/lib.rs

```diff
@@ -347,11 +359,14 @@ impl SIRInterpreter {
         }
     }
 
+    /// XXX unhelpful comment
     /// Implements the Store statement.
     fn store(&mut self, dest: &IPlace, src: &IPlace) {
         self.frames.last_mut().unwrap().mem.store(dest, src);
     }
 
+    /// XXX this might be me being an idiot, but i have no idea what this comment means or, indeed,
+    /// what the function actually does!
```

Comment:
I am pretty sure it implements `dst = &src`.

Reply:
I think that's correct. `mkref` is a SIR and TIR bytecode and this is the implementation of this in the interpreter, right @ptersilie?

Reply:
The fact that three of us aren't 100% sure says that we need, at a minimum, to better document this ;)

Reply:
Yes, it makes a rust reference to a variable, e.g. `&a`. I thought "creates a reference to an iplace" is quite clear, but we can probably improve it.

Reply:
The problem is that "place" is a very vague term, so it doesn't give much of an intuition. I made a comment on the `IPlace` struct too to this effect IIRC.

Reply:
FWIW, `Place` has a very specific meaning in MIR.

Reply:
My impression was that *our* `IPlace` is subtly different?

Reply:
It is, but each MIR place is [lowered to one of our `I[R]Place`s](https://github.com/softdevteam/ykrustc/blob/df02639369734daab3f6505acfc01b8ae783b7be/compiler/rustc_codegen_ssa/src/sir.rs#L497-L502), so it makes sense to maintain that association.

Reply:
But if ours does more, the association implied by the name might be misleading more than it helps?

Reply:
I'm not sure it does more. It does different. It works with byte offsets instead of projections, but conceptually they are the same: an abstract storage location whose actual storage location is to be decided (in their case, by LLVM, in ours, by the trace compiler).

Reply:
OK, maybe `IRPlace` is different-enough-but-similar-enough to keep us all happy :)

Reply:
Improve the doc comment in df14c2a.

Reply:
Fixed in #263.